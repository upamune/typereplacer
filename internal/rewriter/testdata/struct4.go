package sample4

type MyData struct {
	Name string
}
