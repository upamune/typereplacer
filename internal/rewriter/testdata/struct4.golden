package sample4

import "database/sql"

type MyData struct {
	Name sql.Null[string]
}
